src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1314,27 +1314,7 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
             break;
         case NI_Sve_ConditionalSelect:
         {
-            LIR::Use use;
-            bool     foundUse = BlockRange().TryGetUse(node, &use);
-
-            JITDUMP("lowering ConditionalSelect HWIntrinisic (before):\n");
-            DISPTREERANGE(BlockRange(), node);
-            JITDUMP("\n");
-
-            GenTree* cndSelNode = LowerHWIntrinsicCndSel(node);
-
-            JITDUMP("lowering ConditionalSelect HWIntrinisic (after):\n");
-            DISPTREERANGE(BlockRange(), cndSelNode);
-            JITDUMP("\n");
-
-            if (cndSelNode != node)
-            {
-                if (foundUse)
-                {
-                    use.ReplaceWith(cndSelNode);
-                }
-                BlockRange().Remove(node);
-            }
+            LowerHWIntrinsicCndSel(node);
```

Comment:
It looks like `LowerHWIntrinsicCndSel` returns a node, but you're not consuming it